index.bs

```diff
@@ -7300,8 +7300,8 @@ for their contributions as our W3C Team Contacts.
   "EduPersonObjectClassSpec": {
     "publisher": ["Internet2 Middleware Architecture Committee for Education, Directory Working Group (MACE-Dir)"],
     "title": "EduPerson Object Class Specification (200604a)",
-    "href": "https://www.internet2.edu/media/medialibrary/2013/09/04/internet2-mace-dir-eduperson-200604.html",
-    "date": "May 15, 2007"
+    "href": "http://software.internet2.edu/eduperson/",
+    "date": "March 15, 2016"
```

Comment:
When opening https://software.internet2.edu/eduperson/internet2-mace-dir-eduperson-201602.html, the document says: released March 9th 2016 - not March 15th.

Reply:
In an attempt to future-proof, I am suggesting we link to the _directory_ <http://software.internet2.edu/eduperson/> where they list their eduperson schemas. The date I'm suggesting above is the one listed in said directory, in order to help the reader disambiguate items within the directory.  Yes, the date stated _within_ the latest schema doc differs slightly from the doc file's "last modified" date, but <shrug/>.

Reply:
I agree with this - as someone who is part of the 389 directory server project, when we ship the 60eduperson.ldif schemas we comment that it comes from the directory in case there are updates so that we can be made aware of them.

Reply:
That date is the last changed date for the file.  It was posted some days after it was approved. 
While it is not likely to be touched, I can't guarantee that will be the case.  I can ask on the next MACE call what a stable refrence is.

Reply:
> What did you think the problem with using http://software.internet2.edu/eduperson/internet2-mace-dir-eduperson-201602.html as the reference is?

Less brittle to link to the directory, e.g., if/when the schema is updated, as @Firstyear also notes.